frontend/templates/product/sections/ProductSection/index.tsx

```diff
@@ -73,6 +73,19 @@ export function ProductSection({
 
    const isForSale = useIsProductForSale(product);
 
+   const compatibilityDrawerModelsTruncate = 4;
+   const compatibilityDrawerDeviceTruncate = 3;
+   const compatibilityDrawerIncomplete =
+      product.compatibility &&
+      (product.compatibility.devices.length >
+         compatibilityDrawerDeviceTruncate ||
+         product.compatibility.devices
+            .slice(0, compatibilityDrawerDeviceTruncate)
```

Comment:
```suggestion
```

This isn't needed cause if it was longer than `Truncate`, the first part of the `||` would be true and this wouldn't be evaluated.

Reply:
good point